core/textinput/src/Getline.cxx

```diff
@@ -85,6 +85,23 @@ namespace {
    };
    const size_t ROOTTabCompletion::fgLineBufSize = 16*1024;
 
+   class TClingTabCompletion: public TabCompletion {
+   public:
+      TClingTabCompletion() {}
+      virtual ~TClingTabCompletion() {}
+
+      TClingTabCompletion(const TClingTabCompletion&) = delete;
+      TClingTabCompletion& operator=(const TClingTabCompletion&) = delete;
+
+      // Returns false on error
+      bool Complete(Text& line /*in+out*/, size_t& cursor /*in+out*/,
+                    EditorRange& r /*out*/,
+                    std::vector<std::string>& displayCompletions /*out*/) {
```

Comment:
Then I'm confused - I thought we had an `llvm::errs()` that the completions are written to? If so, yes, please change the name, at least here.